testsuite/feeldiscr/test_meshstructured.cpp

```diff
@@ -25,279 +25,247 @@
    \date 2015-04-27
  */
 
-#define USE_BOOST_TEST 1
-#if defined(USE_BOOST_TEST)
 #define BOOST_TEST_MODULE test_meshstructured
 #include <feel/feelcore/testsuite.hpp>
-#endif
 
-#include <feel/feelfilters/loadmesh.hpp>
+#include <feel/feeldiscr/meshstructured.hpp>
+//#include <feel/feelfilters/loadmesh.hpp>
 #include <feel/feeldiscr/pch.hpp>
-#include <feel/feeldiscr/pdh.hpp>
-#include <feel/feelvf/msi.hpp>
+//#include <feel/feeldiscr/pdh.hpp>
+//#include <feel/feelvf/msi.hpp>
 #include <feel/feelvf/integrate.hpp>
 #include <feel/feelvf/form.hpp>
 #include <feel/feelvf/norml2.hpp>
 #include <feel/feelvf/operators.hpp>
 #include <feel/feelvf/operations.hpp>
-#include <feel/feelvf/projectors.hpp>
-#include <feel/feelpoly/multiscalequadrature.hpp>
-#include <feel/feelvf/ginac.hpp>
+//#include <feel/feelvf/projectors.hpp>
+//#include <feel/feelpoly/multiscalequadrature.hpp>
+//#include <feel/feelvf/ginac.hpp>
 #include <feel/feelfilters/exporter.hpp>
-#include <feel/feelfilters/hbf.hpp>
-#if defined(FEELPP_HAS_FFTW)
-#include <fftw3.h>
-#endif
-#include <feel/feeldiscr/meshstructured.hpp>
+//#include <feel/feelfilters/hbf.hpp>
+// #if defined(FEELPP_HAS_FFTW)
+// #include <fftw3.h>
+// #endif
+
+
+using namespace Feel;
+
+FEELPP_ENVIRONMENT_NO_OPTIONS
 
-namespace Feel
-{
 
-inline
-AboutData
-makeAbout()
+#if 0
```

Comment:
the test does not work with coordinates ?

Reply:
the coordinate test is [here](https://github.com/feelpp/feelpp/blob/feature/meshstructured/testsuite/feeldiscr/test_meshstructured.cpp#L213-L269). This comment is a hbf test but it was always commented before my change.